src/sentry/api/event_search.py

```diff
@@ -853,6 +855,23 @@ def to_list(value):
                     kwargs["conditions"].extend(user_conditions)
                 else:
                     kwargs["conditions"].append(user_conditions)
+            elif name == RELEASE_ALIAS:
+                converted_filter = convert_search_filter_to_snuba_query(
+                    SearchFilter(
+                        term.key,
+                        term.operator,
+                        SearchValue(
+                            parse_release(
+                                term.value.value,
+                                params["project_id"],
```

Comment:
This seems like it might break, since `get_latest_release` expects actual project objects, not IDs (`organization_id = projects[0].organization_id`)

Reply:
Poor naming in `get_filter_params` `project_id` is a list of objects, not ids see [here](https://github.com/getsentry/sentry/pull/18568/files#diff-f348b3a3d06653db5b246ec567d39dc4L282)

Worth changing in another PR I think though since that was my first impression of that too

Reply:
Sorry previous comment was completely incorrect, this is more a django thing
```
Release.objects.filter(project__in=[1,2,3])
```

```
projects = [p for p in Project.objects.filter(id__in=[1,2,3])]
Release.objects.filter(project__in=projects)
```

Become the same query with `...WHERE "sentry_release_project"."project_id" IN (1, 2, 3)`

Reply:
Won't this line fail: https://github.com/getsentry/sentry/pull/18568/files#diff-712e897580b290560333dc7b4bb0dd88R229

Since an int does not have an attribute 'organization_id'.

Reply:
Hmm I'll add a check there, but in this case the organization_id is passed from `params` so the function will use that instead of trying to pull it off the projects.

The function is shared with `GroupEventsEndpoint` which only has the list of project objects, but no org which is why it has to grab the org_id